Explorer/Assets/DCL/AvatarRendering/Loading/Assets/AttachmentAssetUtility.cs

```diff
@@ -35,8 +35,12 @@ public static CachedAttachment InstantiateWearable(this IAttachmentsAssetsCache
             {
                 Transform child = children.Value[index];
                 child.gameObject.layer = parent.gameObject.layer;
-            }
 
+                //Wearables shouldnt have animators or animations since it will break the skinning
+                Object.Destroy(child.GetComponent<Animator>());
```

Comment:
Is it safe to delete a component if it is not attached?

Reply:
I had the same doubt. I did a very simple test script, and destroying a `null` reference does not throw an exception. Looks like we are safe

Reply:
The incredible test:

```
        try
        {
            Object.Destroy(GetComponent<Animator>());
        }
        catch (Exception e)
        {
            Debug.Log($"Exception {e}");
        }
```

The GameObject didn't have an animator, and an exception was not thrown